wagtail/users/tests/test_admin_views.py

```diff
@@ -2596,6 +2629,9 @@ def test_get_viewset_cls_with_custom_form(self):
             group_viewset = get_viewset_cls(self.app_config, self.app_config_attr)
         self.assertIs(group_viewset, self.custom_viewset_cls)
         self.assertEqual(group_viewset.icon, "custom-icon")
+        viewset = group_viewset()
+        self.assertIs(viewset.get_form_class(for_update=False), self.create_form_cls)
+        self.assertIs(viewset.get_form_class(for_update=True), self.edit_form_cls)
```

Comment:
I added these assertions to at least confirm that the viewset's `get_form_class()` returns the correct form classes, but I don't know how to test the view themselves...